### ISSUE TITLE
`prisma2 generate` tries to install `@prisma/photon` even in folder without `package.json`

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1147]
I created a project by creating a folder and copying a `prisma/schema.prisma` into it. Then I ran `prisma2 generate` to see what would happen:

```
C:\Users\Jan\Documents\throwaway\modelTest>prisma2 generate
In order to use the "photonjs" generator, you need to install @prisma/photon to your project.
√ Do you want to install it now? ... yes

> @prisma/photon@2.0.0-preview018 postinstall C:\Users\Jan\Documents\node_modules\@prisma\photon
> node scripts/generate.js

npm WARN saveError ENOENT: no such file or directory, open 'C:\Users\Jan\Documents\package.json'
npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Jan\Documents\package.json'
npm WARN Documents No description
npm WARN Documents No repository field.
npm WARN Documents No README data
npm WARN Documents No license field.

+ @prisma/photon@2.0.0-preview018
added 1 package and audited 2 packages in 8.126s
found 0 vulnerabilities

> Downloading Prisma engines for windows [====================] 100%



Generating Photon.js to .\..\..\node_modules\@prisma\photon
Done in 3.76s
```

Seems it found a `node_modules` folder 2 levels up, where it then installed both Photon and also generated the client. Not sure if this is good behaviour.

**Comment by divyenduz** [comment]
@janpio : This is a design decision that we made and I would like to get it changed. It creates `package.json` + `node_modules` inside `prisma/` folder. 

This one almost always gets me with me ending up in a project structure that has 2 `node_modules`/`package.json` files once I set it up manually. 

I would rather fail or have the `cwd()` where we create `package.json`. 

Marking this as a `candidate` for further discussion.
